### ISSUE TITLE
Looking for a new maintainer

### ISSUE THREAD

**Issue by whitequark** [whitequark/ipaddr.js#169]
I haven't had any use for this package for a long time and passive maintenance doesn't seem to be cutting it anymore. I am looking for a new maintainer.

**Comment by josh-cain** [comment]
Hey @whitequark 👋 !  I'd be happy to help, as I'm a downstream consumer of this library.  Let me know what you're looking for and what's involved, happy to chat about it.

**Comment by whitequark** [comment]
Take a look through the recent active issues and let me know what you think and how you'd approach this.

**Comment by josh-cain** [comment]
Looks like many are somewhat old, but I'll start by picking up #65 , as it will afford me a chance to get to know parts of the API I might not be as familiar with.  Will go from there as I get familiar and can pick up more issues 👍 .

**Comment by Uzlopak** [comment]
@whitequark 

I am currently investigating this package for potential performance improvements. And I saw that you search for maintainers. So I was looking in the open issues and saw #167. Maybe it makes sense to explain why you closed AND locked the discussion of #167. Kind of harsh reaction.

I answered the question of the user properly. The dev wanted to know if it is possible to write IPs as numbers. And I wanted to write a function to transform the IPv4 to long integer, and yeah... co-pilot spit that function out. And before I waste my time to implement a high performance solution, for a problem I dont have, i copy pasted it to my comment and credited correctly to co-pilot. (not that somebody thinks, that I use .reduce ;)
But I handcrafted the long integer to IPv4 function which also helps to understand the transformation better, and the explaination on how to store the IP was imho my own thoughts. 

So yeah,it is your realm here. But a little bit more lenience would be nice.
